src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs

```diff
@@ -183,12 +212,15 @@ private void loadBuildingBlocksToModule(Module module, Func<string> getFilename)
             if (filename.IsNullOrEmpty())
                return;
 
-            var items = loadBuildingBlock(buildingBlockType, filename);
+            var listOfNewBuildingBlocks = loadBuildingBlock(buildingBlockType, filename);
```

Comment:
would rename the function _loadBuildingBlock_ to plural _loadBuildingBlock**s**_